### DISCUSSION TITLE
How to find the corresponding host by ident?

### DISCUSSION THREAD

**Discussion by tianwen2019** [ccfos/nightingale#1672] [Q&A]
有个奇怪的告警，找不到主机了，目前知道的信息只有ident，默认是hostname，这个该如何查找呢

**Comment by UlricQin** [comment]
新版本可以根据来源IP查一下：
<img width="1359" alt="image" src="https://github.com/ccfos/nightingale/assets/2769466/8a5edc88-e387-4b9e-9b4b-144402b8036b">
